ruby/files_and_serialization/project_event_manager.md

```diff
@@ -1105,8 +1094,8 @@ code.
 Instead of including the template within our application, we will instead load
 the template using the same File tools we used at the beginning of the exercise.
 
-* Create a file named 'form_letter.html' in the root of your project directory.
-* Copy the html template defined above into that file and save it.
+- Create a file named 'form_letter.HTML' in the root of your project directory.
```

Comment:
Built-in rule fixing this only because it's not an inline code block. Using backticks keeps it happy.

```suggestion
- Create a file named `form_letter.html` in the root of your project directory.
```